source/common/http/http3/conn_pool.cc

```diff
@@ -65,10 +65,13 @@ allocateConnPool(Event::Dispatcher& dispatcher, Random::RandomGenerator& random_
       host, priority, dispatcher, options, transport_socket_options, random_generator, state,
       [&quic_stat_names,
        &scope](HttpConnPoolImplBase* pool) -> ::Envoy::ConnectionPool::ActiveClientPtr {
+        pool->addLog(Logger::Logger::debug, "Create Http/3 client");
```

Comment:
nit: How about "Create" => "Creating" and below "Fail" => "Failed"?

Reply:
done